src/pages/index.page.jsx

```diff
@@ -17,13 +17,16 @@ export default function Home() {
                 we&#39;re adding new components frequently!
               </p>
             </div>
-            <div>
+            <picture className="home-top__top-image">
+              <source
+                srcset="/images/home-header-image_dark.svg"
+                media="(prefers-color-scheme: dark)"
+              />
               <img
-                className="home-top__top-image"
                 src="/images/home-header-image_light.svg"
                 alt="Illustration of dog surrounded by accessible components"
               />
-            </div>
+            </picture>
```

Comment:
The layout of this section has changed, and I wrapping this `picture` in a `div` will fix that. I would also put the `className` back on the `img` tag instead of the `picture` element.

The `picture` element lets us do cool stuff like loading alternate images based on media queries, but the `img` is effectively what is rendered.

Reply:
Makes sense, I made these changes. Thanks! :)